services/supervisord/maintail.go

```diff
@@ -26,13 +26,19 @@ type eventType string
 
 const (
 	// mirror http://supervisord.org/subprocess.html#process-states
-	stopped          eventType = "STOPPED"
-	stopping         eventType = "STOPPING"
-	starting         eventType = "STARTING"
-	running          eventType = "RUNNING"
-	exitedExpected   eventType = "EXITED (expected)"
-	exitedUnexpected eventType = "EXITED (unexpected)"
-	fatal            eventType = "FATAL"
+
+	// Stopping indicates that stop of a process was requested.
+	Stopping eventType = "STOPPING"
```

Comment:
If we will remove watching supervisord events in dbaasClient we can revert these changes.

Reply:
Reverted.